### ISSUE TITLE
EzTemplate object has no close function

### ISSUE THREAD

**Issue by durack1** [UV-CDAT/uvcdat#1585]
The `EzTemplate` object has no `close` or `exit` functions, so I am uncertain how to purge or manipulate an `EzTemplate` object during an interactive session once it is created (and previewed):

```python
>>> Canvas = EzTemplate.Multi(rows=2,columns=2)
>>> Canvas.preview('Template4x4',bg=bg)
>>> dir(Canvas)
['__class__', '__delattr__', '__doc__', '__format__', '__getattribute__', '__hash__', '__init__', '__module__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__slots__', '__str__', '__subclasshook__', '_columns', '_getcolumns', '_getrows', '_gettemplate', '_rows', '_setcolumns', '_setrows', '_settemplate', '_template', 'clean', 'columns', 'get', 'id', 'legend', 'list', 'margins', 'preview', 'rows', 'spacing', 'template', 'template_names', 'x']
>>> Canvas.clean()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/uvcdat/2.4.0rc1/lib/python2.7/site-packages/vcsaddons/Multi.py", line 229, in clean
    self.x.removeP(t)
  File "/usr/local/uvcdat/2.4.0rc1/lib/python2.7/site-packages/vcs/Canvas.py", line 1275, in removeP
    return self.canvas.removeP(*args)
AttributeError: 'module' object has no attribute 'removeP'
```
During the session above, a `VCS` canvas window appears, however I am unable to close this using the `X` (top right corner) or using a `Canvas.close()` (or equivalent, that I can find) on the commandline.

**Comment by doutriaux1** [comment]
it shouldn't. When you do a `get` you get a template that you can remove. Will fix the clean issue.
By the way you do ``NOT`` get a canvas back. it's an EZTemplate object, not a vas canvas.

**Comment by durack1** [comment]
@doutriaux1 ok, so I have a persistent `Canvas/EzTemplate` object that I cannot close.. So how do I go about closing it using a `get`?

**Comment by doutriaux1** [comment]
you mean after a preview?

**Comment by durack1** [comment]
@doutriaux1 exactly..

**Comment by doutriaux1** [comment]
yes after the preview it is a canvas, you can access it as `Canvas.x.close()` in your case. You're right `Canvas.preview()` should probably return an handle to the canvas.

**Comment by doutriaux1** [comment]
will put this in the bug fix. Will become something like that:
```
Multi = EzTemplate.Multi(rows=2,columns=2)
Canvas = Multi.preview('Template4x4',bg=bg)
Canvas.close()
```

**Comment by durack1** [comment]
@doutriaux1 ok so `Canvas.x.close()` works (the window terminates), but I had no idea that I needed to call the `x` to initiate a `close`.. The above looks much more reasonable to me..

**Comment by durack1** [comment]
@doutriaux1 also, the inability to `close` it using the window controls (top right hand `X`) is also a concern to me.. These controls should always work..

**Comment by durack1** [comment]
@doutriaux1 fixed #1591?

**Comment by doutriaux1** [comment]
yes
